src/commands/dev.ts

```diff
@@ -148,6 +149,17 @@ export async function command(commandOptions: CommandOptions) {
     }
   }
 
+  function addToDependents(spec: string, fileLoc: string) {
```

Comment:
`fileLoc` should be `fileUrl` or `fileUrlPath`.

This codebase uses the `*Loc` suffix to mean a full, absolute file path on disk.